libs/stocks/data-access-price-query/src/lib/+state/price-query.effects.ts

```diff
@@ -24,12 +24,30 @@ export class PriceQueryEffects {
       run: (action: FetchPriceQuery, state: PriceQueryPartialState) => {
         return this.httpClient
           .get(
-            `${this.env.apiURL}/beta/stock/${action.symbol}/chart/${
-              action.period
-            }?token=${this.env.apiKey}`
+            `${this.env.apiURL}/beta/stock/${action.symbol}/chart/max?token=${this.env.apiKey}`
```

Comment:
getting data for 15 years and then filtering the data we need between two dates.
If there is a function which could pull the data between two dates on the api, that would be better option.
I assumed that there is no predefined method on the api. Totally understand that this is an expensive call.
We could get 5 years or 2 years data and then filter but I have used max though